security/appsec/security-intro/what-else-do-you-need-to-know.md

```diff
@@ -41,10 +41,14 @@ Before diving into this skill, you should make sure you have a strong foundation
 
 If you'd like to learn more about application security, you should make sure you have a strong foundation in the following:  
 
-At least one scripting language (even bash, check out our other topics)  
-???  
-Operating Systems  
-How memory in a computer works, and where data is located at any given time  
+**At least one scripting language (even bash, check out our other topics)**
+
+???
+
+**Operating Systems**
+
+**How memory in a computer works, 
+and where data is located at any given time**
```

Comment:
I think these should go on the same line
```suggestion
**How memory in a computer works, and where data is located at any given time**
```

Reply:
Ok, I applied your suggestion